### DISCUSSION TITLE
How to use?

### DISCUSSION THREAD

**Discussion by davidmacalister** [dominicegginton/bento#1] [Q&A]
As it uses this splash screen, I don't know how I can use it in my browser and the README doesn't say anything.

**Comment by dominicegginton** [comment]
@DavidMacalister
Thanks for giving **Bento** a try. I have updated the [README](https://github.com/dominicegginton/bento#install) with some installation instructions for Google Chrome. And also plan to add an extra build step to package the built extension bundle making the installation process for both Firefox and Safari much more seamless, as they both don't support loading unpacked extensions locally.
